source/manual/alerts/email-alert-api-high-queue-latency.html.md

```diff
@@ -14,10 +14,44 @@ review_in: 6 months
 This means the time it takes for a job to be processed is unusually high. This
 may indicate a problem down the line which is preventing workers from being
 processed. It may also imply the threshold is too low if a large number of
-emails have being sent out due to a content change.
+emails are currently being processed.
+
+#### `delivery_transactional`
+
+The `delivery_transactional` queue is used for when we need to send one time
+emails. Some examples of one time emails could be a confirmation of a
+subscription or when validating a new subscribers email address.
+
+The [Sidekiq priority][priority] for this queue is weighted highly and there
+should only be a very small number of jobs on the queue at any one time which
+means that these emails should be processed pretty promptly. If this is not the
+case and this alert is firing then we should investigate.
+
+We should initially look for degradation of the service as high latency
+normally indicates degraded performance or high load, or both.
+
+Some diagnostic steps you could take are below:
+
+* Check to see if we have a high number of emails queued up. If these is a
```

Comment:
This is a bit of a non-step: it's the definition of the alert. One useful thing we could do here, or in the intro, is point people to the specific Grafana dashboard/panel they can see this on.

Reply:
Sure, I've linked to the relevant graphs.